pyproject.toml

```diff
@@ -6,13 +6,13 @@ authors = ["MrPowers <matthewkevinpowers@gmail.com>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0"
+python = ">=3.8,<4.0"
```

Comment:
3.8 is EOL very soon too, let's drop it?

Reply:
Good point, however I think it's good to keep supporting it until it's officially EOL, unless something is really blocking us from improving this project further. Are there any issues, e.g. spark compatibility issues?

Reply:
No, no known issues. Ok, let's wait until official EOL!